### PR TITLE
Update Challenge Description 03_01

### PR DESCRIPTION
**Pull request by abolinsky**
Accompanying EN one [here](https://github.com/ms1-learner/python-02-en/pull/2)

### CODE REVIEW

File: 03_inheritance_and_polymorphism/01_challenge/README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 抽象基底クラス、継承、ポリモーフィズムを使用して、ポリモーフィックな性質を備えた初歩的な車両システムを作成します。
 
 ## 要件
-- `Vehicle` を抽象クラスとして定義し、`get_details()` 具象メソッドと `start_engine()` 抽象メソッドを追加します。
-- 車両には`make` (メーカー)、`model` (モデル)、`year` (年式) のインスタンス変数が必要です。
+- `Vehicle` を抽象クラスとして定義します。`Vehicle` には`make` (メーカー)、`model` (モデル)、`year` (年式) のインスタンス変数が必要です。
+- 各車両タイプの詳細情報を含めるように `Vehicle` に `get_details()` 抽象メソッドを追加します。
```
Comment (dancer-yuma) [inline 03_inheritance_and_polymorphism/01_challenge/README.md:6]:
```suggestion
- 各車両タイプの詳細情報を返すよう `Vehicle` に `get_details()` 抽象メソッドを追加します。
```

**Review by dancer-yuma — COMMENTED** [review]
@abolinsky 
Just one suggestion!

File: 03_inheritance_and_polymorphism/01_challenge/README.md (hunk 1 above)
Comment (abolinsky) [reply 03_inheritance_and_polymorphism/01_challenge/README.md]: Accepted

**Review by dancer-yuma — APPROVED** [review]
@abolinsky 
Looks good to me!

**Pull request merged**